students/sendecki_andrzej/lesson_06_dicts_tuples_sets_args_kwargs/list_to_dictionary_function_for_fantasy_game_inventory.py

```diff
@@ -0,0 +1,21 @@
+def displayInventory(inventory):
+    print("Inventory:")
+    item_total = 0
+    for k, v in inventory.items():
+        print(str(v) + " " + k)
+        item_total += v
+    print("Total number of items: " + str(item_total))
+
+def addToInventory(inventory, addedItems):
+    for item in addedItems:
+        default_value = inventory.get(item,0)
+        if item in inventory:
+            inventory[item] = default_value + 1
+        else:
+            inventory.setdefault(item, default_value + 1)
```

Comment:
Good. 
It's also worth to know about defaultdict https://www.accelebrate.com/blog/using-defaultdict-python/
and
`.get()` method http://www.tutorialspoint.com/python/dictionary_get.htm